viewer/src/gojsTemplates.ts

```diff
@@ -49,4 +49,4 @@ export type NodeDataArrayTemplate = StateMachineTemplate | StateTemplate | LinkL
 export interface DrawComponentData {
     nodeDataArray: Array<NodeDataArrayTemplate>;
```

Comment:
nodeDataArray is a redundant name (with class name and member type)
nodes could be a better choice.